src/common/modules/UnicodeTransformationHandler.js

```diff
@@ -46,11 +46,14 @@ export function transformText(text, transformationId) {
 export function getTransformationType(transformationId) {
     if (transformationId.startsWith(CASE_ID_PREFIX)) {
         return TRANSFORMATION_TYPE.CASING;
-    } else if (transformationId.startsWith(CODE_CASE_ID_PREFIX)) {
+    }
+    if (transformationId.startsWith(CODE_CASE_ID_PREFIX)) {
         return TRANSFORMATION_TYPE.CODE_CASING;
-    } else if (transformationId.startsWith(FONT_ID_PREFIX)) {
+    }
+    if (transformationId.startsWith(FONT_ID_PREFIX)) {
```

Comment:
hmm personally I liked the old way more, but well just a nit pick…

Reply:
This change came from the [noUselessElse rule](https://biomejs.dev/linter/rules/no-useless-else/) in Biome, which is similar but more aggressive than the [no-else-return rule](https://eslint.org/docs/latest/rules/no-else-return) in ESLint. The "else" is useless here due to the return in the previous statement.